app/templates/survey/new_question.html

```diff
@@ -77,49 +76,55 @@ <h2 class="ui header">
             </tbody>
         </table>
 
+        <button class="ui button" id="button-copy">Add Response</button>
+
+        <div class="ui hidden divider"></div>
+
         {{ f.render_form_field(form.submit) }}
         {{ f.end_form(form) }}
 
     </div>
 </div>
 
 <script type="text/javascript">
+    $(".next-question-skip").hide();
```

Comment:
Maybe I'm just not remembering how JS works lmao but is this supposed to be outside of `$(document).ready()`?